java/ql/src/Security/CWE/CWE-925/ImproperIntentVerification.qhelp

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE qhelp PUBLIC
+  "-//Semmle//qhelp//EN"
+  "qhelp.dtd">
+
+<qhelp>
+
+<overview>
+<p>
+When an android application uses a <code>BroadcastReciever</code> to receive intents, 
+it is also able to receive explicit intents that are sent directly to it, regardless of its filter.
```

Comment:
What does the `its` pronoun refer to here, the app or the `BroadcastReceiver` (sorry, not a developer)?

Reply:
It refers t the BroadcastReceiver